spec/controllers/user_targets_controller_spec.rb

```diff
@@ -145,6 +145,23 @@
         end
       end
     end
+
+    context 'when there are too many targets created' do
+      let(:new_target) { target_params('title', 50, topic.id, 0, 0) }
+
+      before(:each) do
+        FactoryGirl.create_list(:user_target,10, user: user, topic: topic, latitude: 0, longitude: 0)
```

Comment:
Add these attributes to the factory and remove the unnecessary ones from the `create_list` call